tests/rms/test_generate_petro_jobs_for_field_update.py

```diff
@@ -0,0 +1,404 @@
+"""Run tests in RMS using rmsapi to generate jobs for petrophysics module
+
+Creates a tmp RMS project in given version which is used as fixture
+to test the script generate_petro_jobs_for_field_update.py
+
+
+This requires a ROXAPI license, and to be ran in a "roxenvbash" environment; hence
+the decorator "roxapilicense"
+
+"""
+
+import contextlib
+import copy
+import filecmp
+import shutil
+from os.path import isdir
+from pathlib import Path
+
+import pytest
+import xtgeo
+
+with contextlib.suppress(ImportError):
+    import roxar  # type: ignore
+    import roxar.jobs  # type: ignore
+
+from fmu.tools.rms.generate_petro_jobs_for_field_update import (
+    check_rms_project,
+    create_new_petro_job_per_facies,
+    define_new_variable_names_and_correlation_matrix,
+    read_specification_file,
+    write_petro_job_to_file,
+)
+
+# ======================================================================================
+# settings to create RMS project!
+REMOVE_RMS_PROJECT_AFTER_TEST = True
+
+TMPD = Path("TMP")
+TMPD.mkdir(parents=True, exist_ok=True)
+
+
+PROJNAME = "tmp_project_generate_petro_jobs.rmsxxx"
+PRJ = str(TMPD / PROJNAME)
+RESULTDIR = TMPD / "jobs"
+RESULTDIR.mkdir(parents=True, exist_ok=True)
+
+REFERENCE_DIR = Path("rms/generate_jobs_testdata")
```

Comment:
I needed to add `tests` here to make it work ... `Path("tests/rms/generate_jobs_testdata")`

Reply:
Fixed